src/modules/zerion-api/zerion-api.ts

```diff
@@ -43,20 +43,23 @@ function getZpiHeaders() {
   };
 }
 
+type UrlInput = { endpoint: string } | { url: string | URL };
```

Comment:
Do we need this endpoint case? I can't find it in the code

Reply:
Updated other methods to use this